src/transform.js

```diff
@@ -24,10 +24,30 @@ var proj4 = require('proj4');
  */
 //////////////////////////////////////////////////////////////////////////////
 
+var transformCache = {};
+/* Up to maxTransformCacheSize squared might be cached.  When the maximum cache
+ * size is reached, the cache is completely emptied.  Since we probably won't
+ * be rapidly switching between a large number of transforms, this is adequate
+ * simple behavior. */
+var maxTransformCacheSize = 10;
+
 var transform = function (options) {
```

Comment:
> Do you mean a generic cache for things besides transform

Yes generic cache object since we have been caching multiple things now and I feel like we could use something like that (not for this PR though).